src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -40,10 +45,14 @@
   public Label loginErrorLabel;
   public Pane loginRoot;
 
+
   @Inject
-  public LoginController(UserService userService, PreferencesService preferencesService) {
+  public LoginController(UserService userService, PreferencesService preferencesService, PlatformService platformService, @Value("${login.createAccountUrl}") String createUrl, @Value("${login.forgotLoginUrl}") String forgotLoginUrl) {
```

Comment:
Can you refactor this to use the new way; Inject `ClientProperties` and read it from there?